CHANGES.txt

```diff
@@ -44,14 +44,19 @@ Changes
 Fixes
 =====
 
+ - Added proper handling when memory requirements of a distributed query
```

Comment:
I'd rephrase this a bit. Only JOIN queries were really affected (or at least, that's the only place where this adds an improvement)